packages/web/src/pages/collection-page/CollectionPageProvider.tsx

```diff
@@ -307,12 +307,11 @@ class CollectionPage extends Component<
       }
     }
 
-    // check if a track has been added to collection
-    if (
-      metadata &&
-      prevMetadata &&
-      metadata.track_count > prevMetadata.track_count
-    ) {
+    const currentTrackCount = this.props.trackCount
+    const previousTrackCount = prevProps.trackCount
```

Comment:
we usually prefer destructuring

I can see skipping it here though since you have to rename anyway